### PR TITLE
(chore) add location category links to start page

### PR DESCRIPTION
**Pull request by dushan-madetech**
We want users of the service to land on the start page and be able to
easily search for jobs within a county or london borough. To achieve
this we added links to the start page via a partial.

## Screenshots of UI changes:

Screenshot 2019-12-04 at 16 44 59

### CODE REVIEW

**Comment by dushan-madetech** [comment]
@adrianclay @cesidio @Renny4Real Which bits of text do you think should be internationalised?

File: app/views/pages/home/_location_categories.html.haml [hunk 1]
```diff
@@ -1 +1,14 @@
-%h1.govuk-heading-xl Location Categories
+%h1.govuk-heading-xl Explore teaching jobs by location
```
Comment (cesidio) [inline app/views/pages/home/_location_categories.html.haml]: This text should be i18n

File: app/views/pages/home/_location_categories.html.haml [hunk 2]
```diff
@@ -1 +1,14 @@
...
+.govuk-grid-row
+  .govuk-grid-column-one-half
+    %h2.govuk-heading-m English counties
+    %ul.govuk-list
+      - LocationCategory.counties.each do | county |
+        %li
+          = link_to county, location_category_path(county), class: 'govuk-link'
+  .govuk-grid-column-one-half 
+    %h2.govuk-heading-m London boroughs
```
Comment (cesidio) [inline app/views/pages/home/_location_categories.html.haml]: This text should be i18n

File: app/views/pages/home/_location_categories.html.haml [hunk 3]
```diff
@@ -1 +1,14 @@
-%h1.govuk-heading-xl Location Categories
+%h1.govuk-heading-xl Explore teaching jobs by location
+.govuk-grid-row
+  .govuk-grid-column-one-half
+    %h2.govuk-heading-m English counties
```
Comment (cesidio) [inline app/views/pages/home/_location_categories.html.haml]: This text should be i18n

File: app/views/pages/home/_location_categories.html.haml (hunk 1 above)
Comment (dushan-madetech) [reply app/views/pages/home/_location_categories.html.haml]: Done 👍

File: app/views/pages/home/_location_categories.html.haml (hunk 2 above)
Comment (dushan-madetech) [reply app/views/pages/home/_location_categories.html.haml]: Done 👍

File: app/views/pages/home/_location_categories.html.haml (hunk 3 above)
Comment (dushan-madetech) [reply app/views/pages/home/_location_categories.html.haml]: Done 👍

File: app/views/pages/home/_location_categories.html.haml [hunk 4]
```diff
@@ -1 +1,14 @@
-%h1.govuk-heading-xl Location Categories
+%h1.govuk-heading-xl #{ I18n.t('pages.home.location_links_heading') }
+.govuk-grid-row
+  .govuk-grid-column-one-half
+    %h2.govuk-heading-m #{ I18n.t('pages.home.counties') }
```
Comment (cesidio) [inline app/views/pages/home/_location_categories.html.haml]: this should be
```
t('.counties')
```
and the translation should be put in
```
pages.home.location_categories.counties
```

File: app/views/pages/home/_location_categories.html.haml [hunk 5]
```diff
@@ -1 +1,14 @@
-%h1.govuk-heading-xl Location Categories
+%h1.govuk-heading-xl #{ t('.location_links_heading') }
```
Comment (cesidio) [inline app/views/pages/home/_location_categories.html.haml]: this should be
`%h1.govuk-heading-l` not xl

Comment (dushan-madetech) [reply app/views/pages/home/_location_categories.html.haml]: Done 👍

File: app/views/pages/home/_location_categories.html.haml (hunk 4 above)
Comment (dushan-madetech) [reply app/views/pages/home/_location_categories.html.haml]: resolvedaroo 👍

**Pull request merged**